monitor/api/src/main/java/org/datacleaner/monitor/scheduling/model/ScheduleDefinition.java

```diff
@@ -43,7 +43,8 @@
     private boolean _distributedExecution;
     private String _dateForOneTimeSchedule;
 	private Map<String,String> _jobMetadataProperties;
-    private boolean _runOnHadoop; 
+    private boolean _runOnHadoop;
+    private String _hotFolder;
```

Comment:
Because it actually has changed (either I or my IDE removed an empty space from the end of the line).